docker/examples/elastic/ES/Dockerfile

```diff
@@ -49,10 +51,16 @@ RUN wget https://raw.githubusercontent.com/geopython/pygeoapi/master/tests/load_
 RUN echo "xpack.security.enabled: false" >> config/elasticsearch.yml
 RUN echo "http.host: 0.0.0.0" >> config/elasticsearch.yml
 
+RUN yum install -y python3
+
 RUN yum --enablerepo=extras -y install epel-release \
-	&& yum install -y python-pip \
-	&& yum install -y python-setuptools \
-	&& pip install --upgrade pip && pip install elasticsearch \
+	&& yum install -y python3-pip \
```

Comment:
cc @jorgejesus 

I think each `RUN` command creates an extra 'layer'?  Maybe we can update to:

```
RUN yum --enablerepo=extras -y install epel-release \
        && yum install -y python3 python3-pip python3-setuptools python-typing \
        && pip install --upgrade pip elasticsearch elasticsearch-dsl \
	&& yum clean packages
```

Note the addition of `elasticsearch-dsl` since CQL was added earlier today.

Reply:
Yes, you are completely right. I separated the commands to  facilitate the debug. Let me put them back!

Reply:
@tomkralidis https://github.com/geopython/pygeoapi/pull/739/commits/457b07ab2a057d9473facdb83f12377d38c13ae5